test/Wheel-test.js

```diff
@@ -27,6 +27,7 @@ describe('Wheel', function() {
     wheel.spin()
     expect().to.equal();
     //Maybe spy on this to see if it is triggering a dom update? 
```

Comment:
You don't have to test for updates to the DOM.

Reply:
We should look into how spies work a little bit more. Let's do a little bit of independent research and then discuss this again :)